packages/material-ui-codemod/src/v5.0.0/private-imports.test/actual.js

```diff
@@ -0,0 +1,9 @@
+import MenuItem from '@material-ui/core/MenuItem';
+import Tab from '@material-ui/core/Tab';
+import MuiTabs from '@material-ui/core/Tabs';
+import createTheme from '@material-ui/core/styles/createTheme';
+import yellow from "@material-ui/core/colors/yellow";
+// just testing that these get ignored
+import TableContext from '@material-ui/core/Table/TableContext';
+import TabScrollButton from '@material-ui/core/Tabs/TabScrollButton';
+import SwitchBase from '@material-ui/core/internal/SwitchBase';
```

Comment:
Not sure if these should be ignored or not, but I've added them as they were there in the v4.0.0/optimal-imports. I've listed potential solutions to this in the PR description.